packages/kbn-expandable-flyout/src/components/preview_section.tsx

```diff
@@ -139,7 +139,7 @@ export const PreviewSection: React.FC<PreviewSectionProps> = ({
       <EuiSplitPanel.Outer
         css={css`
           margin: ${euiTheme.size.xs};
-          box-shadow: 0 0 4px 4px ${euiTheme.colors.darkShade};
+          box-shadow: 0 0 16px 0px rgb(105, 112, 125, 0.5);
```

Comment:
I don't think we should be using rbg like this, we should instead find the matching color within eui, or light and dark mode won't work properly.
Also using `rgba` would be more accurate than `rgb` here.

Reply:
good point on dark mode, reverted back to eui theme colors, we should used the `transparentize` function to handle transparency too.